Incidents/incident-5.md

```diff
@@ -0,0 +1,165 @@
+# Incident 5
+
+|                            |             |
+| -------------------------: | :---------- |
+| **Reported on**            | July-12-2022 |
+| **Mitigation**             | July-12-2022 |
+| **Solution Completed**     | July-15-2022 |
+| **Severity**               | SEV-3        |
+
+
+## What happened? 
+
+After updating **peer-ec1** and **peer-ap1** from Catalyst `v4.8.0` to `v4.8.4` the nodes started to throw CPU alarms and the Lambdas Service entered in a continuous reboot cycle. 
+ 
+## Why did it happen?
+
+After several tests, it was detected that the change causing this issue was a [refactor](https://github.com/decentraland/catalyst/pull/1168) on the Lambdas routes to use Well Known Components but the rollout converged with a major DB change that led to think the issue was there and this situation added complexity to the troubleshooting process. 
+
+## Issue Timeline 
+
+Several changes were accumulated in the release rollout and those potential sources of the issues added complexity to finding out what was the root cause of the problem.  
+
+July-12-2022 - New Catalyst version [v4.8.4](https://github.com/decentraland/catalyst/releases/tag/4.8.4) created. This version had a major change in the DB schema and all the queries impacted by this.
+July-12-2022 - Upgrade nodes **peer-ec1** and **peer-ap1** from `v4.8.0` to `v4.8.4`
+July-12-2022 - After a while of the version update, alarms started to be triggered and lambdas services started to reboot
+July-12-2022 - To mitigate the issue **peer-ec1** was rollback to `v4.8.0` and the `maxUsers` of **peer-ap1** was set to ``0`` to avoid users being sent to the node
```

Comment:
```suggestion
July-12-2022 - To mitigate the issue **peer-ec1** was rolled back to `v4.8.0` and the `maxUsers` of **peer-ap1** was set to ``0`` to avoid users being sent to the node

```